master/internal/command/command.go

```diff
@@ -38,460 +37,245 @@ import (
 // terminated state in the master before garbage collecting.
 const terminatedDuration = 24 * time.Hour
 
-// terminateForGC is an internal message indicating that the command actor
-// should stop and garbage collect its state.
-type terminateForGC struct{}
-
 // queueStates are allocation states which the API and UI will show as "Queued".
 var queueStates = []model.AllocationState{
 	model.AllocationStatePending,
 	model.AllocationStateAssigned,
 }
 
-func enrichState(state model.AllocationState) taskv1.State {
-	if slices.Contains(queueStates, state) {
-		return taskv1.State_STATE_QUEUED
-	}
-	return state.Proto()
-}
+// Command is executed in a containerized environment on a Determined cluster.
+// Locking in: toNTSC, startCmd, onExit, deleteIfInWorkspace.
```

Comment:
nit: since we are locking these functions it might be best to capitalize them as it signifies they are being called externally.

Reply:
capitalized!